.github/workflows/release.yml

```
@@ -240,8 +240,51 @@ jobs:
           -Netsuke ./target/debug/netsuke.exe
           -Manifest ./tests/data/windows-recipe-smoke.yml
 
+  release-admission-canaries:
+    needs: metadata
+    runs-on: namespace-profile-netsuke
+    permissions:
+      actions: read
+      contents: read
+    env:
+      NETSUKE_RELEASE_ADMISSION_METRICS_FILE: >-
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)